lib/Module/KCallable.cpp

```diff
@@ -0,0 +1,46 @@
+//===-- KCallable.cpp -----------------------------------------------------===//
+//
+//                     The KLEE Symbolic Virtual Machine
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#include "klee/Module/KCallable.h"
+
+#include "llvm/ADT/Twine.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/InlineAsm.h"
+#include "llvm/IR/LLVMContext.h"
+
+using namespace llvm;
+using namespace klee;
+
+unsigned KCallable::globalAsmId = 0;
+
+KCallable::KCallable(Function *func) : func(func), isFunc(true) {}
+KCallable::KCallable(InlineAsm *asmValue)
+    : asmValue(asmValue), asmId(globalAsmId++), isFunc(false) {
+  asmName = "__asm__" + Twine(asmId).str();
+}
+
+StringRef KCallable::getName() const {
+  if (isFunc)
+    return func->getName();
+  return asmName;
```

Comment:
Could you please replace this if statement with a conditional expression for consistency with the rest of methods in `KCallable`?